.dockerignore

```diff
@@ -12,4 +12,3 @@ __pycache__/
 *$py.class
 
 .venv
```

Comment:
minor detail, this also needs to be changed in the template repository then right?

Reply:
What is the problem in this section ?